google/cloud/aiplatform/utils/prediction_utils.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from typing import Optional, Sequence
+
+from google.cloud.aiplatform.constants import prediction
+
+
+def get_prediction_aip_http_port(
+    serving_container_ports: Optional[Sequence[int]] = None,
+):
+    """Gets the used prediction container port from serving container ports.
+
+    When you create a Model, set the containerSpec.ports field. The first entry in this
+    field becomes the value of AIP_HTTP_PORT. Default value is 8080.
```

Comment:
I was confused by this description, then I realized this is exactly what our public documentation says, haha.

Nit, I think it reads better as -

If containerSpec.ports was specified during Model (LocalModel?) creation time, retrieve the first entry in this field. Otherwise use the default value of 8080. The env var AIP_HTTP_PORT will be set to this value.

Reply:
Done.